lib/glific/jobs/user_jobs.ex

```diff
@@ -86,7 +86,7 @@ defmodule Glific.Jobs.UserJob do
   def list_user_jobs(args \\ %{}) do
     args
     |> Repo.list_filter_query(UserJob, &Repo.opts_with_name/2, &filter_with/2)
-    |> Repo.all()
+    |> Repo.all(skip_organization_id: true)
   end
 
```

Comment:
just wanted to understand how skipping the organization_id is helping move the jobs to a success state?

Reply:
`list_user_jobs` fetch the user_jobs with the given filters (in this case, `%{filter: %{status: "pending", all_tasks_created: true}}`). After fetching the pending jobs we move it to success state. if we are not skipping the organization_id sometimes it won't fetch all the pending jobs across org_ids.